README.md

```diff
@@ -9,218 +8,20 @@ Nextcloud server packaged as a snap. It consists of:
 - Redis 6
 - mDNS for network discovery
 
+* [**Hardware requirements**](https://github.com/nextcloud-snap/nextcloud-snap/wiki/Hardware-requirements)
+* [**OS requirements**](https://github.com/nextcloud-snap/nextcloud-snap/wiki/OS-requirements)
 
-## How to install
+## Installation
+Find instructions on how to install Snap and Nextcloud-snap in the [Snapstore](https://snapcraft.io/nextcloud). However Snap will be pre-installed on the latest Ubuntu versions.
```

Comment:
"snap" is not actually a thing, so talking about how it's pre-installed on Ubuntu is confusing. "snapd" is what you're looking for here, but even that I think it not useful information to provide here, since in general it's not something folks need to worry about. It's not our job to teach people about snaps anyway-- I recommend focusing on our product: the Nextcloud snap. I would like to include the release channel information here as well, otherwise they won't have that information unless they stumble on it in the wiki, and might ultimately install the wrong thing.

For this section, I recommend this:

````suggestion
## How to install

[![Get it from the Snap Store](https://snapcraft.io/static/images/badges/en/snap-store-white.svg)](https://snapcraft.io/nextcloud)

Type the following in your terminal:

```
$ sudo snap install nextcloud
```

By default you'll get the latest stable release, and it will automatically update to subsequent stable releases, but there are [other releases available as well](https://github.com/nextcloud/nextcloud-snap/wiki/Release-strategy).
````